modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/disruptor/delivery/ConcurrentContentReadTaskBatchProcessor.java

```diff
@@ -31,9 +31,10 @@
 import org.apache.commons.logging.LogFactory;
 import org.wso2.andes.kernel.AndesMessageMetadata;
 
-
 import java.util.ArrayList;
```

Comment:
@a5anka I couldn't change this class to an event handler. Is it possible to change this to an event handler as well?
not an urgent or priority though.